parm/mom6/MOM_input_template_025

```diff
@@ -98,7 +98,7 @@ WRITE_GEOM = 2                  ! default = 1
                                 ! If =0, never write the geometry and vertical grid files. If =1, write the
                                 ! geometry and vertical grid files only for a new simulation. If =2, always
                                 ! write the geometry and vertical grid files. Other values are invalid.
-SAVE_INITIAL_CONDS = True      !   [Boolean] default = False
+SAVE_INITIAL_CONDS = False      !   [Boolean] default = False
```

Comment:
Is turning this off intentional? I feel like this gets accidentally changed every time.

Reply:
Yes, this file should match what is here: https://github.com/ufs-community/ufs-weather-model/blob/Release/P8b/tests/parm/MOM_input_template_025

Reply:
Okay, we have just historically been changing this for g-w so that we save those ICs. If we no longer need to modify it, we can just take it out of the g-w and link to the UFS copy.

Reply:
@jiandewang can you please confirm that this was changed with your awareness? Or if I messed this setting up please confirm and I'll revert.

Reply:
@jiandewang commended on the main thread.  I'm going to advocate for keeping this as is.

Reply:
I have no problem with that. I was just pointing out until now we had always changed it for g-w. If we don't need to make modifications anymore, we can (in a separate commit) replace these local versions with links to the ones that live in UFS.